apis/v1alpha1/generator.yaml

```diff
@@ -43,6 +43,8 @@ resources:
         is_immutable: true
       CustomKeyStoreID:
         is_immutable: true
+      EnableKeyRotation:
+        type: bool
```

Comment:
nit: We could also directly infer the type with something similar to what we have in ecr-controller:https://github.com/aws-controllers-k8s/ecr-controller/blob/main/generator.yaml#L12-L15
```yaml
EnableKeyRotation:
    operation: EnableKeyRotation
    path: KeyId
```

Reply:
@A-Hilaly I tried this. But, imo, `EnableKeyRotation` should be `boolean`. And when I tested this, it was giving `*string` (ref image).
Hence, I have put as `boolean` explicitly.

<img width="936" alt="image" src="https://user-images.githubusercontent.com/6952642/197538478-ae3e176c-457c-4592-8898-6a32b37f2525.png">

Reply:
Ohhh i see, makes sense!